db/seeds/db_populator.rb

```diff
@@ -243,6 +239,28 @@ def create_cases(casa_org, options)
         )
       end
 
+      random_case_contact_count.times do
+        create_case_contact(new_casa_case)
+      end
+
+      # guarantee at least one case contact before and after most recent past court date
+      most_recent_past_court_date = most_recent_past_court_date(new_casa_case.id)
```

Comment:
It can be a little confusing to name a variable as the same name as the method (and can cause headaches later when you try to call the method, only it has been overwritten). I would suggest just changing line 248 to:

```
if most_recent_past_court_date(new_casa_case.id)
  ...
end
```

Reply:
Yes I did wonder about this approach. I will change the variable name!